libs/payments/ui/src/lib/client/components/Header/index.tsx

```diff
@@ -0,0 +1,372 @@
+                      mobileLandscape:h-auto mobileLandscape:drop-down-menu mobileLandscape:top-10 mobileLandscape:-start-52 desktop:-start-50"
+              >
+                <div className="flex flex-wrap">
+                  <div className="flex w-full py-4 gap-2 items-center flex-col tablet:w-auto tablet:relative">
+                    <button onClick={() => setBentoMenuRevealed(false)}>
+                      <Image
+                        src={closeIcon}
+                        alt="Close"
+                        width="16"
+                        height="16"
+                        className="absolute top-5 end-5 mobileLandscape:hidden fill-current"
+                      />
+                    </button>
+                    <div className="mt-12 px-8 text-center mobileLandscape:mt-0">
+                      <Localized id="next-bento-menu-tagline">
+                        <h2>
+                          More products from Mozilla that protect your privacy
+                        </h2>
+                      </Localized>
+                    </div>
+                    <div className="w-full text-xs">
+                      <ul className="list-none">
+                        <li>
+                          <LinkExternal
+                            href={links.desktop}
+                            className="block p-2 ps-6 hover:bg-grey-100"
+                          >
+                            <div className={iconClassNames}>
+                              <Image src={desktopIcon} alt="" />
+                            </div>
+                            <Localized id="next-bento-menu-firefox-desktop">
+                              Firefox Browser for Desktop
+                            </Localized>
+                          </LinkExternal>
+                        </li>
+                        <li>
+                          <LinkExternal
+                            href={links.mobile}
+                            className="block p-2 ps-6 hover:bg-grey-100"
+                          >
+                            <div className={iconClassNames}>
+                              <Image src={mobileIcon} alt="" />
+                            </div>
+                            <Localized id="next-bento-menu-firefox-mobile">
+                              Firefox Browser for Mobile
+                            </Localized>
+                          </LinkExternal>
+                        </li>
+                        <li>
+                          <LinkExternal
+                            href={links.monitor}
+                            className="block p-2 ps-6 hover:bg-grey-100"
+                          >
+                            <div className={iconClassNames}>
+                              <Image src={monitorIcon} alt="" />
+                            </div>
+                            <Localized id="next-bento-menu-monitor-3">
+                              Firefox Browser for Mobile
```

Comment:
`Firefox Browser for Mobile` -> `Mozilla Monitor` 👀

Reply:
thaaaaank you